axelrod/tournament.py

```diff
@@ -376,4 +376,4 @@ def __init__(self, players, edges, match_generator=SpatialMatches,
 
         self.edges = edges
         self.match_generator = SpatialMatches(
-            players, turns, self.game, repetitions, edges)
+            players, turns, self.game, repetitions, self.noise, edges)
```

Comment:
Could you add an assertion in the init test that checks that when a spatial tournament is initialised with noise, it's match generator has the same noise value?

```
tournament = axelrod.SpatialTournament(...)
self.assertEqual(tournament.match_generator.noise, noise) 
```

Something like that.